Automattic/wp-calypso
# Stats: Extract Emails module into wrapper component

## Proposed Changes

To keep the module consistent (and avoid drift) while we work on the Subscribers page, consolidate the Emails module into a new wrapper component as `StatsEmailModule`.

If we agree to this change, should be merged after #75379.

## Testing Instructions



1. Launch the Calypso Live link and visit Stats → Traffic.
2. Confirm the Emails module is visible at the bottom of the modules grid.
3. Add `?flags=stats/subscribers-section` to the URL and refresh.
4. Click the new Subscribers link in the navigation header.
5. Confirm the Emails module is visible here as well.

## Pre-merge Checklist
